pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -1,28 +1,33 @@
-{ stdenv, fetchurl, pkgconfig, glib, freetype, cairo, libintl
+{ stdenv, fetchFromGitHub, pkgconfig, glib, freetype, cairo, libintl
+, meson, ninja
 , gobject-introspection
 , icu, graphite2, harfbuzz # The icu variant uses and propagates the non-icu one.
 , ApplicationServices, CoreText
 , withCoreText ? false
 , withIcu ? false # recommended by upstream as default, but most don't needed and it's big
 , withGraphite2 ? true # it is small and major distros do include it
-, python
+, python3
 }:
 
 let
-  version = "2.6.7";
+  version = "2.7.1";
   inherit (stdenv.lib) optional optionals optionalString;
+  boolMesonFlag = opt: b:
```

Comment:
`enabled` and `disabled` are not of `bool` type but of `feature` type in Meson.

Reply:
Renamed for clarity.